### DISCUSSION TITLE
The admin screen cannot be displayed normally and the screen is blank.

### DISCUSSION THREAD

**Discussion by fenrir-liutiecheng** [mozilla/hubs#3162] [Hubs Cloud Questions] [answered]
I  follow the wizard( https://hubs.mozilla.com/docs/hubs-cloud-aws-quick-start.html) to create the HubsCloud server. At this time, there is no problem with the admin screen.
[normal image](http://chuantu.xyz/t6/741/1602583732x-1404795709.png)

But when I folow the wizard(https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html), used the latest master code without any modification. 
[deploy image](http://chuantu.xyz/t6/741/1602583857x1700340450.png)

After redeploying, the admin screen cannot be displayed normally and the screen is blank.
[error image](http://chuantu.xyz/t6/741/1602583901x-591238492.png)

Did I do something wrong?

**Comment by robinkwilson** [comment, marked as answer]
Are you using the "master" branch or "hubs-cloud" branch on the hubs repo? In the https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html instructions, it tells you to base off of hubs-cloud branch. **Do not base off of the "master" branch.**

Can you give us any errors in the browser's developer tools > Console tab OR Network tab? Those would help.

If you run `npm run undeploy` does the admin panel render again normally -- so we can isolate the problem to the custom client?

**Reply by fenrir-liutiecheng** [reply]
https://fenrir-hub.com
https://fenrir-hub.com/admin
Please see these two addresses above

**Reply by fenrir-liutiecheng** [reply]
Console error:
The source list for Content Security Policy directive 'connect-src' contains an invalid source: 'wss://:80'. It will be ignored.
The source list for Content Security Policy directive 'connect-src' contains an invalid source: 'https://:80'. It will be ignored.
